src/common/mfu_param_path.c

```diff
@@ -418,6 +419,7 @@ void mfu_param_path_check_copy(uint64_t num, const mfu_param_path* paths,
     /* just have rank 0 check */
     if(rank == 0) {
         /* count number of readable source paths */
+/*
```

Comment:
We'll want to find a DAOS equivalent for this I think, since it changes how the tools operate.